BackendNew/controllers/students.js

```diff
@@ -214,6 +214,26 @@ const BasicStudent = function(student) {
   this.password = student.password;
 };
 
+// Student model.
+const Student = function(student) {
+  this.createdBy = student.createdBy;
+  this.roleName = "Student";
+  this.email = student.email;
+  this.fullName = student.fullName;
+  this.nameWithInitials = student.nameWithInitials;
+  this.indexNumber = student.indexNumber;
+  this.password = student.password;
+  this.phoneNumber = student.telephoneNumber;
+  this.Sem1GPA = student.gpa.first_sem;
+  this.Sem2GPA = student.gpa.second_sem;
+  this.Sem3GPA = student.gpa.third_sem;
+  this.Sem4GPA = student.gpa.fourth_sem;
+  this.SGPA = student.gpa.SGPA;
+  this.PreferedArea1 = student.PreferedArea1;
+  this.PreferedArea2 = student.PreferedArea2;
+  this.PreferedArea3 = student.PreferedArea3;
+};
+
```

Comment:
Isn't it good to have separate folder for models?

Reply:
Yeah. Will move.

Reply:
@hasinichathu there is a seperate folder for models? then what should we do?

Reply:
That means?? We can simply import the model when necessary. I saw several places where same model is used

Reply:
Yeah. Will make a separate file for models